src/interfaces/resource-interfaces/special-resolution-sample-form-interface.ts

```diff
@@ -0,0 +1,8 @@
+/** Interface to define the special resolution sample form. */
+
+export interface SpecialResolutionSampleFormIF {
+    header?: string
+    text?: string
+    label?: string
+    path?: string
+}
```

Comment:
Looks good. For now it's specific to Special Resolutions. If this changes then we can look again what to call this or whatever.